README.md

```diff
@@ -1226,12 +1226,16 @@ hints for the pairwise grouping with comments or empty lines.
     * in functions:
         * `f`, `g`, `h` - function input
         * `n` - integer input usually a size
-        * `index` - integer index
+        * `index`, `i` - integer index
         * `x`, `y` - numbers
+        * `xs` - sequence
+        * `m` - map
         * `s` - string input
+        * `re` - regular expression
```

Comment:
Totally missed this, but shouldn't be surprised given the names of the functions themselves. :-)